tests/test_web_urldispatcher.py

```diff
@@ -515,3 +516,17 @@ async def test_static_absolute_url(aiohttp_client, tmpdir) -> None:
     client = await aiohttp_client(app)
     resp = await client.get("/static/" + str(fname))
     assert resp.status == 403
+
+
+async def test_decoded_match_regex(aiohttp_client) -> None:
+    app = web.Application()
+
+    async def handler(_):
+        return web.Response()
+
+    app.router.add_get("/{user_ids:(([0-9]+)|(u_[0-9a-f]{16}))(,(([0-9]+)|(u_[0-9a-f]{16})))*}/hello", handler)
```

Comment:
Looks like this needs to be formatted for us to see the test run in the CI: https://github.com/aio-libs/aiohttp/pull/5633/checks?check_run_id=2384352929#step:6:120.

Reply:
done